src/services/catalog.data.ts

```diff
@@ -104,6 +106,30 @@ export async function deletePhysicalItemFromCatalog(catalog_id: string): Promise
     });
 }
 
+export async function putPhysicalItemInCatalog(issue: CatalogNewspaperEditDto): Promise<void> {
+  const token = await getKeycloakTekstToken();
+
+  return fetch(`${process.env.CATALOGUE_API_PATH}/newspapers/items`, {
```

Comment:
Ser det ligger en rewrite på katalog-kall i next.config.mjs som er benyttet noen steder, men ikke alle. Ikke en case for denne tasken å fikse men noterer det bare uansett.